config/app.php

```diff
@@ -221,6 +222,7 @@
         'Form' => Collective\Html\FormFacade::class,
         'Html' => Collective\Html\HtmlFacade::class,
         'Laratrust'   => Laratrust\LaratrustFacade::class,
+        'GeoIP' => \Torann\GeoIP\Facades\GeoIP::class,
```

Comment:
@c0repwn3r and this needs to be `'GeoIP' => Torann\GeoIP\Facades\GeoIP::class,`

Reply:
Fixed